deb/brave-keyring/DEBIAN/postinst

```diff
@@ -2,3 +2,9 @@
 
 # Delete (expired) key added by old install instructions
 apt-key --keyring /etc/apt/trusted.gpg del D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821
+
+# Delete the old keyring file if the new one is already in use.
+if grep -q '^[ ]*deb.*signed-by=/usr/share/keyrings/brave-browser-release.asc.*brave-browser-apt-release.s3.brave.com' \
+    /etc/apt/sources.list /etc/apt/sources.list.d/*; then
```

Comment:
This isn't especially pretty, but I couldn't find a more proper way to do this. None of the `apt` tools return information about `signed-by`.

Also, is it ok to use `grep` in postinst if grep itself is not a dependency of the package? Not sure.

Reply:
For simplicity, I'd suggest only looking at `/etc/apt/sources.list.d/brave-browser*` since that's what our instructions say. Anybody not following the instructions carefully gets to do this cleanup themselves.